bin/ch-run.c

```diff
@@ -466,13 +468,20 @@ static error_t parse_opt(int key, char *arg, struct argp_state *state)
       args->seccomp_p = true;
       break;
 #endif
+   case -15: // --set-env0
+      parse_set_env(args, arg, '\0');
+      break;
    case -16: // --warnings
       for (int i = 1; i <= parse_int(arg, false, "--warnings"); i++)
          WARNING("this is warning %d!", i);
       exit(0);
       break;
-   case -15: // --set-env0
-      parse_set_env(args, arg, '\0');
+   case -17: // --logging-test
+         if (arg == NULL) {
+            logging_print(false);
+         } else {
+            logging_print((!strcmp(arg, "fail")));
```

Comment:
Does this mean all arguments other than `fail` are accepted but a no-op? That seems a bit user-unfriendly, but it might be fine for a test option.